client/src/components/Diagrams.js

```diff
@@ -0,0 +1,135 @@
+import BoardDashboard from "components/BoardDashboard"
+import { InputField } from "components/FieldHelper"
+import Fieldset from "components/Fieldset"
+import Messages from "components/Messages"
+import { FastField, Formik } from "formik"
+import PropTypes from "prop-types"
+import React, { useState } from "react"
+import { Button, Modal } from "react-bootstrap"
+import utils from "utils"
+import * as yup from "yup"
+import "./Diagrams.css"
```

Comment:
To avoid clutter, generic diagramming components might be placed in a sub-directory `src/components/diagrams/`.